django/contrib/auth/models.py

```diff
@@ -75,9 +75,9 @@ class Meta:
 
     def __str__(self):
         return "%s | %s | %s" % (
-            six.text_type(self.content_type.app_label),
-            six.text_type(self.content_type),
-            six.text_type(self.name))
+            self.content_type.app_label,
+            str(self.content_type),
+            self.name)
```

Comment:
Might want to move the `)` to the next line and add a trailing comma when you are around.